sdkutil/object_id_test.go

```diff
@@ -156,86 +156,112 @@ func TestGroupIDString(t *testing.T) {
 func TestBlockReflectionID(t *testing.T) {
 	t.Run("deployment-id", func(t *testing.T) {
 		id := testutil.DeploymentID(t)
-		block := sdkutil.ReflectBlockID(id)
 
-		assert.Equal(t, id.Owner, *block.Owner)
-		assert.Equal(t, id.DSeq, *block.DSeq)
+		t.Run("reflect", func(t *testing.T) {
+			block := sdkutil.ReflectBlockID(id)
```

Comment:
result of `sdkutil.ReflectBlockID` is always not nil. Does it make sense to return result as value? Will help a bit for escape analysis

Reply:
Done! Tests updated.